### PR TITLE
updated text track unit tests to use full es6 syntax

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
Currently a few of the unit tests are using partial es6 or no es6 at all. This Pull Request seeks to rectify that.

## Specific Changes proposed
* converted all text-track test files to full es6 syntax
* fixed most issues with `vjsstandard` (where it would not break jshint)

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: test/unit/tracks/text-track-list.test.js [hunk 1]
```diff
@@ -2,33 +2,33 @@ import TextTrackList from '../../../src/js/tracks/text-track-list.js';
 import TextTrack from '../../../src/js/tracks/text-track.js';
 import EventTarget from '../../../src/js/event-target.js';
 
-var noop = Function.prototype;
-var genericTracks = [
+const noop = () => {};
```
Comment (dmlap) [inline test/unit/tracks/text-track-list.test.js]: Looks like you actually made this unused now.

**Comment by dmlap** [comment]
LGTM

File: test/unit/tracks/tracks.test.js [hunk 2]
```diff
@@ -406,19 +408,19 @@ test('removes cuechange event when text track is hidden for emulated tracks', fu
     'texttrackchange should be not be called since mode is hidden');
 });
 
-test('should return correct remote text track values', function () {
+test('should return correct remote text track values', function() {
   let fixture = document.getElementById('qunit-fixture');
+  let html = '';
```
Comment (gkatsev) [inline test/unit/tracks/tracks.test.js]: can we change this to a template string?

**Comment by gkatsev** [comment]
LGTM.

**Pull request closed without merging**